src/main/java/dk/cngroup/lentils/controller/HintController.java

```diff
@@ -49,40 +56,58 @@ public String deleteHint(@PathVariable("hintId") final Long hintId) {
         Hint hint = hintService.getHint(hintId);
         Long cypherId = hint.getCypherId();
         hintService.deleteById(hintId);
-        return REDIRECT_HINT_LIST + "?cypherId=" + cypherId;
+        return REDIRECT_HINT_LIST + CYPHERID_PARAMETER + cypherId;
     }
 
     @GetMapping(value = "/update/{hintId}")
     public String addForm(@PathVariable("hintId") final Long hintId, final Model model) {
```

Comment:
Mohl by jsi prosim sjednotit/zmenit nazvy metod `addForm` a `newHint`? V jednom se vyskytuje substring "String" a ve druhe ne a taky je blbe to, ze tato metoda `addForm`  je na endpointu "/update/{hintId}", takze by tam melo byt spis slovo `update` nez `add`.